packages/alfa-css/src/value/unit.ts

```diff
@@ -16,22 +16,40 @@ export namespace Unit {
     /**
      * {@link https://drafts.csswg.org/css-values/#relative-lengths}
      */
-    export type Relative =
-      | "em"
-      | "ex"
-      | "ch"
-      | "rem"
-      | "vw"
-      | "vh"
-      | "vmin"
-      | "vmax";
+    export type Relative = Relative.Font | Relative.Viewport;
+
+    export namespace Relative {
+      /**
+       * {@link https://drafts.csswg.org/css-values/#font-relative-lengths}
+       */
+      export type Font = "em" | "ex" | "ch" | "rem";
+
+      /**
+       * {@link https://drafts.csswg.org/css-values/#viewport-relative-lengths}
+       */
+      export type Viewport = "vw" | "vh" | "vmin" | "vmax";
+    }
 
     export function isRelative(input: string): input is Relative {
+      return isFontRelative(input) || isViewportRelative(input);
```

Comment:
I think in this (and the others), `unit` would be a more descriptive variable name than `input`.